lib/ii.c

```diff
@@ -8379,6 +8442,7 @@ grn_ii_column_update(grn_ctx *ctx, grn_ii *ii, grn_id rid, unsigned int section,
   grn_obj *old_, *old = oldvalue, *new_, *new = newvalue, oldv, newv;
   grn_obj buf, *post = NULL;
 
+  GRN_SLOW_LOG_PUSH(ctx, GRN_LOG_DEBUG);
```

Comment:
This may be unbalanced because of `return ctx->rc`.

Reply:
Fixed. (but not elegant...)